assignment/scripts/array-practice.js

```diff
@@ -76,19 +88,39 @@ console.log('The animals are now', animalArray);
 // 4.d TODO: Remove the food at the beginning of your array & 
 //     log both the food removed and the updated array
 
+removedFood = foodsArray.shift();
+console.log('Removed the first food', removedFood);
+console.log('My favorite foods are now', foodsArray);
+
 // 4.e (STRETCH) TODO: Replace the second food in your array
 //      with another one of your favorite foods.
 //      Then log the updated array.
 
+foodsArray [1] = 'pasta';
+console.log('My favorite foods are now:', foodsArray);
+
 // 4.f (STRETCH) TODO: Sort your favoriteFoods array
 //     in reverse alphabetical order. Log the array.
 
+const reversed = foodsArray.reverse();
+console.log('My favorite foods in reversed:', reversed);
+
 // 4.g (STRETCH) TODO: Convert your array to a string
 //     putting the word "and" between each item.
 //     eg "tacos and pizza and pasta". Log the string.
 
+const elements = foodsArray;
+
+console.log(elements.join('-'));
```

Comment:
You used the right array method, but the prompt was to put `" and "` between the foods, not `"-"`!